src/core/group/IdentifiableUnique.h

```diff
@@ -70,6 +75,8 @@ class IdentifiableUnique {
     std::size_t sizeOf() const;
     template<typename T2>
     std::vector<const T2*> getOf() const;
+    template<typename T2>
+    std::vector<const T*> filterOf() const;
```

Comment:
Maybe we can think a better name

Reply:
After some ideas came out, `getByCastingToParent` would do the trick, as you suggested.